src/components/GrowlNotification/index.tsx

```diff
@@ -16,15 +19,26 @@ const INACTIVE_POSITION_Y = -255;
 
 const PressableWithoutFeedback = Pressables.PressableWithoutFeedback;
 
-function GrowlNotification(_, ref) {
+function GrowlNotification(_: unknown, ref: ForwardedRef<GrowlRef>) {
     const translateY = useRef(new Animated.Value(INACTIVE_POSITION_Y)).current;
     const [bodyText, setBodyText] = useState('');
     const [type, setType] = useState('success');
-    const [duration, setDuration] = useState();
+    const [duration, setDuration] = useState<number | undefined>();
     const theme = useTheme();
     const styles = useThemeStyles();
 
-    const types = {
+    type GrowlIconTypes = Record<
+        string,
```

Comment:
Can you add a comment on what the key of the `GrowlIconTypes` represents?

Reply:
I've added a comment to clarify that part of the code.